Tasks/PublishBuildArtifacts/publishbuildartifacts.ts

```diff
@@ -5,6 +5,14 @@ import tl = require('vsts-task-lib/task');
 
```

Comment:
Eric, this .js file is not added for these as well as other tasks beforel. Are you sure we want to add it?